templates/admin/group/group_row.html.ep

```diff
@@ -0,0 +1,60 @@
+<li
+    id="<%= $is_parent ? 'parent' : 'job' %>_group_<%= $group->id %>"
```

Comment:
you have so many is_parents in here that I almost believe having 2 templates is easier to read - the duplicated code seems rather limited